### PR TITLE
projects/lwip: initial integration

### PR DESCRIPTION
**Pull request by neuschaefer**
Add lwIP (Lightweight TCP/IP stack).

TODO before merging:

- [ ] finalize discussion at https://savannah.nongnu.org/bugs/index.php?64010
- [ ] add initial corpora

### CODE REVIEW

File: projects/lwip/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# Copyright 2023 J. Neuschäfer
```
Comment (jonathanmetzman) [inline projects/lwip/Dockerfile:1]: Need to give google the copyright. If you really don't want to do this, then put most of the code elsewhere and just have the code in this repo pull that code down and call it.

File: projects/lwip/project.yaml [hunk 2]
```diff
@@ -0,0 +1,12 @@
+homepage: https://savannah.nongnu.org/projects/lwip/
+language: c
+primary_contact: "Simon Goldschmidt <goldsimon@gmx.de>"
```
Comment (jonathanmetzman) [inline projects/lwip/project.yaml:3]: just use an email address, no brackets or spaces or your name please.

**Comment by jonathanmetzman** [comment]
Let me know when this is ready and I'll ask the committee if we want to add this project.

File: projects/lwip/Dockerfile (hunk 1 above)
Comment (neuschaefer) [reply projects/lwip/Dockerfile:1]: Ok

File: projects/lwip/project.yaml (hunk 2 above)
Comment (neuschaefer) [reply projects/lwip/project.yaml:3]: ok

**Comment by neuschaefer** [comment]
Ah sorry, I completely forgot about this PR. I might pick it up later, but I currently don't have plans to do so.

**Pull request closed without merging**